{"name": "test_002", "status": "failed", "statusDetails": {"message": "AssertionError: assert 1 == 100", "trace": "def test_002():\n        print(\"-----------test_002--------------\")\n>       assert 1==100\nE       assert 1 == 100\n\ntestcase\\test_demo01.py:15: AssertionError"}, "attachments": [{"name": "stdout", "source": "a448d452-5531-4a10-8752-5799f6bfe76a-attachment.txt", "type": "text/plain"}], "start": 1690421609432, "stop": 1690421609432, "uuid": "c9b66dbd-137b-41b1-8d0f-fbb71c95d340", "historyId": "ac0be3a25232c3d3dd61b983a9dad760", "testCaseId": "39b0d78c484a8bcbacfb079d993b73e6", "fullName": "testcase.test_demo01#test_002", "labels": [{"name": "parentSuite", "value": "testcase"}, {"name": "suite", "value": "test_demo01"}, {"name": "host", "value": "zhanghao"}, {"name": "thread", "value": "12196-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.test_demo01"}]}